share/spice/hex/content.handlebars

```diff
@@ -0,0 +1,8 @@
+<p><strong>{{releases.0.version}}</strong> is the latest of <strong>{{releases.length}}</strong> releases and it was published <em>{{updated_at}}</em>.</p>
```

Comment:
@moollaza I don’t know any service like Hound CI or Codacy that supports Perl either.

In other open source projects the approach that I’ve seen and follow, usually using Travis CI, is the following:

* Install the dependencies, normally these dependencies are cached.
* Retrieve, build, test the main code base, in this case, Perl (you already do this)
* Lint and build the SASS/LESS/CSS code with some Gulp/Grunt/Brunch task manager.
* Lint and build the JS code with some Gulp/Grunt/Brunch task manager.

The problem with this approach is that the process runs over the all code base. So, if at this stage, suppose that your source code does not comply with a certain rule, for sure will report false positives in some Pull Request (PR) and this could create some confusion to the developers because their PR will fail. Also, it can be difficult and risky to rewrite a lot of code just to comply with some style guide, but it will necessary to rewrite some code for certain errors. So, at the end, I don’t have a proposed solution, sorry for that  :(